Tests/scripts/collect_tests/utils.py

```diff
@@ -171,8 +171,13 @@ def __init__(self, path: Path):
     @property
     def _has_no_id(self):
         # some content files may not have an id
-        return self.path.name == 'pack_metadata.json' or self.path.name.endswith(
-            '_schema.json') or self.path.name.endswith('testdata.json')
+        file_path_splitted = self.path.parts
+        return self.path.name == 'pack_metadata.json' \
+            or self.path.name.endswith('_schema.json') \
+            or self.path.name.endswith('testdata.json') \
```

Comment:
```suggestion
            or self.path.name.endswith('_schema.json') \
            or self.path.name == 'testdata.json' \
```

Reply:
That's how it was before.
I'm not sure the file is called testdata.json. It could be a suffix.
I prefer not to change it.